modules/era/lua_dynamis/globals/era_dynamis_spawning.lua

```diff
@@ -1969,18 +1971,22 @@ xi.dynamis.mobOnEngaged = function(mob, target)
             mob:SetMagicCastingEnabled(false)
             mob:SetMobAbilityEnabled(false)
 
-            mob:addStatusEffectEx(xi.effect.BIND, xi.effect.BIND, 1, 3, 3, 0, 0, 0, xi.effectFlag.NO_LOSS_MESSAGE)
+            mob:addStatusEffectEx(xi.effect.BIND, xi.effect.BIND, 1, 3, 6, 0, 0, 0, xi.effectFlag.NO_LOSS_MESSAGE)
             mob:timer(3000, function(mobArg)
                 if mob:isAlive() then
                     xi.dynamis.spawnDynamicPet(target, mob, mobArg:getMainJob())
                     mobArg:entityAnimationPacket("shsm")
                     mobArg:SetAutoAttackEnabled(true)
                     mobArg:SetMagicCastingEnabled(true)
                     mobArg:SetMobAbilityEnabled(true)
+
+                    if mobArg:hasStatusEffect(xi.effect.BIND) then
+                        mobArg:delStatusEffectSilent(xi.effect.BIND)
+                    end
                 end
             end)
-        elseif mob:getMainJob() == xi.job.DRG then
-            mob:useMobAbility(xi.jsa.CALL_WYVERN)
+        -- elseif mob:getMainJob() == xi.job.DRG then
```

Comment:
Do mobs use their 2hr to summon the wyvern? is that why this is commented out?

Reply:
Yeah DRG mobs shouldn't have their wyvern out until they 2hr in dyna